dev-docs/apis.adoc

```diff
@@ -0,0 +1,81 @@
+= APIs in Solr
+:toc: left
+
+Solr's codebase currently has a handful of ways of defining APIs.
+This complexity stems largely from two ongoing transitions:
+1. Away from our "v1" APIs and towards the new "v2" API. 
+2. Away from a legacy API framework towards the off-the-shelf JAX-RS library for implementing our v2 APIs
+
+As these transitions complete, this complexity should simplify considerably.
+But in the interim, this document can help guide developers who need to understand or modify APIs in Solr.
+
+== API Types
+
+APIs in Solr (regardless of whether v1 or v2) can typically be classified as either a "per-core" or an "admin" API.
```

Comment:
a --> as

Reply:
Not quite sure what word you were suggesting that correction for, but I've tried to reword the sentence a bit to make it scan a little more cleanly.